sky/adaptors/seeweb.py

```diff
@@ -1,6 +1,10 @@
 """ Seeweb Adaptor """
 import configparser
 from pathlib import Path
+from typing import Optional
+
+from pydantic import ValidationError
```

Comment:
style nit: import modules instead of classes. https://google.github.io/styleguide/pyguide.html#22-imports